Sources/SwiftBuild/SWBBuildParameters.swift

```diff
@@ -43,6 +43,8 @@ public struct SWBRunDestinationInfo: Codable, Sendable {
     public var supportedArchitectures: [String]
     public var disableOnlyActiveArch: Bool
     public var hostTargetedPlatform: String?
+    public var sdkManifestPath: String?
```

Comment:
We need to determine what happens when multiple of these are specified. What happens if you specify sdkManifestPath _and_ sdk _and_ sdkVariant? Can you provide some examples of what the values would be in such cases?

Similarly, what happens if targetArchitecture and triple are both specified? What are some examples of inputs?